examples/gateway/huggingface/readme.md

```diff
@@ -0,0 +1,16 @@
+## Example route configuration for Huggingface Text Generation Inference
```

Comment:
Can this example be expanded to actually provide a self-contained runnable example? 

There are quite a few setup steps required to get this server running and it would be beneficial to show how to actually get started with this provider. 

For reference, please see the very similar self-hosted example guide for using MLflow's inference capabilities to run an sbert sentence-transformers implementation: https://github.com/mlflow/mlflow/blob/master/examples/gateway/mlflow_serving/README.md 

The guide should show each required step to get up and running with the example. There should also be some sort of a disclaimer about the hardware considerations to use flash attention / paged attention and CUDA requirements in the system running the container (as these considerations are the very reason to use this framework and require some amount of system configuration to do before the example would properly show off the benefits of TGI).

Reply:
Great changes!! Thank you :)